src/applications/vre/28-1900/containers/VeteranInformation.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * The Challenge:
+ * Logged out view allows a veteran to fill out name, dob, ssn, and va file number.
+ * Logged in view pulls data from the profile to populate a view only component with name, dob, and gender.
+ *
+ * The Implications
+ * There needs to be a flexible schema that can support a logged out flow where we collect personal information
+ * but also allows for the limited information we pull from the profile if they are logged in.
+ *
+ * There needs to be a mechanism that toggles between these two states (logged in/out).
+ * We need to connect to the store in this component.
+ *
+ * Done:
+ * 1. Toggle between view component and form fields when logged in/out
+ * 2. Update formData via setData action dispatch
+ */
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
```

Comment:
This component shows a static view component if a user is logged in and form fields if they are logged out. There will be future tickets to refactor this. For now, it's functional.